contributing/background/glossary.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Glossary
+nav_order: 3
+parent: Background
+grand_parent: Contributing
+---
+# Glossary of Common Terms
+
+This page contains a selection of key terms important to understanding the structure of data within DataCommons.
+
+## Term List
+
+**Statistical variable**
```

Comment:
I might propose `statistical variable ([StatisticalVariable](https://datacommons.org/browser/StatisticalVariable))` -- the intention is to make sure users unify the concept of a statistical variable with Data Commons' StatisticalVariable type. Do you think that is valuable? I might be stuck in my world..

Reply:
Great idea! I've incorporated it into the most recent